src/FrontDoor/FrontDoor/Cmdlets/NewAzureRmFrontDoorWafManagedRuleExclusionObject.cs

```diff
@@ -0,0 +1,75 @@
+﻿// ----------------------------------------------------------------------------------
+//
+// Copyright Microsoft Corporation
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+// http://www.apache.org/licenses/LICENSE-2.0
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+// ----------------------------------------------------------------------------------
+
+using System.Linq;
+using System.Management.Automation;
+using Microsoft.Azure.Commands.FrontDoor.Common;
+using Microsoft.Azure.Commands.FrontDoor.Models;
+using Microsoft.Azure.Commands.ResourceManager.Common.ArgumentCompleters;
+
+namespace Microsoft.Azure.Commands.FrontDoor.Cmdlets
+{
+    /// <summary>
+    /// Defines the New-AzFrontDoorWafManagedRuleExclusionObject cmdlet.
+    /// </summary>
+    [Cmdlet("New", ResourceManager.Common.AzureRMConstants.AzureRMPrefix + "FrontDoorWafManagedRuleExclusionObject"), OutputType(typeof(PSManagedRuleExclusion))]
+    public class NewAzureRmFrontDoorWafManagedRuleExclusionObject : AzureFrontDoorCmdletBase
+    {
+        /// <summary>
+        /// Exclusion match variable
+        /// </summary>
+        [Parameter(Mandatory = true, HelpMessage = "Match variable")]
+        [PSArgumentCompleter("RequestHeaderNames", "RequestCookieNames", "QueryStringArgNames", "RequestBodyPostArgNames")]
+        public string Variable { get; set; }
+
+        /// <summary>
+        /// Exclusion operator
+        /// </summary>
+        [Parameter(Mandatory = true, HelpMessage = "Operator")]
+        [PSArgumentCompleter("Equals", "Contains", "StartsWith", "EndsWith", "EqualsAny")]
```

Comment:
what does `equalsany` mean? variable equals any of the selectors?

Reply:
Added a comment